### PR TITLE
Add changes to com.ibm.ws.org.apache.cxf.ws.security.2.6.2 into Open-Liberty

### PR DESCRIPTION
**Pull request by BerksanAtes**
for #9765
Bundle wired to use CXF trough internal bundles. Plus open-liberty specific changes

### CODE REVIEW

**Comment by BerksanAtes** [comment]
#run-libby-bot

File: dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+-buildpath: \
+    org.apache.cxf:cxf-rt-ws-security;strategy=exact;version=${cxfVersion}, \
+    com.ibm.ws.org.apache.cxf.cxf.api.2.6.2, \
+    com.ibm.ws.org.apache.cxf.cxf.rt.core.2.6.2, \
+    com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.2.6.2, \
+    com.ibm.ws.org.opensaml.opensaml.2.6.1, \
+    com.ibm.ws.org.apache.ws.security.wss4j.1.6.7, \
+    com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2, \
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd:35]: this is using the new version of rt.ws.policy; it needs to be 2.6.2.

File: dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#*******************************************************************************
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-ws-security;2.6.2;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
+
+globalize: false
+instrument.ffdc: true
+
+cxfVersion=2.6.2
+
+-includeresource: \
+    @${repo;org.apache.cxf:cxf-rt-ws-security;${cxfVersion};EXACT}!/!META-INF/maven/*|\
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd:19]: @${repo;org.apache.cxf:cxf-rt-ws-security;${cxfVersion};EXACT}!/!META-INF/maven/*|\ is the | at the end of the line valid? There is no comma either. If the | character is valid then it should be maven/*| , \

File: dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    META-INF/cxf/bus-extensions.txt=override/META-INF/cxf/bus-extensions.txt, \
+    cxf-ehcache.xml=resources/cxf-ehcache.xml, \
+    org/apache/cxf/ws/security/policy/interceptors/Messages.properties=resources/org/apache/cxf/ws/security/policy/interceptors/Messages.nlsprops, \
+    org/apache/cxf/ws/security/policy/model/Messages.properties=resources/org/apache/cxf/ws/security/policy/model/Messages.nlsprops, \
+    org/apache/cxf/ws/security/trust/Messages.properties=resources/org/apache/cxf/ws/security/trust/Messages.nlsprops, \
+    org/apache/cxf/ws/security/wss4j/Messages.properties=resources/org/apache/cxf/ws/security/wss4j/Messages.nlsprops
+
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd:26]: It doesn't look like your including the Open-Liberty CXF source once it's compiled. The last line of  -includeresources header should be org/apache/cxf=${bin}/org/apache/cxf

File: dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides [hunk 4]
```diff
@@ -26,13 +29,14 @@ Import-Package:  \
   com.ibm.websphere.ras, \
   com.ibm.websphere.ras.annotation, \
   com.ibm.ws.ffdc, \
+  net.sf.ehcache.*, \
   *
 
 Include-Resource: \
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides:35]: The include-resources header should only exist in the bnd.bnd file.

File: dev/com.ibm.ws.org.apache.cxf.ws.security/bnd.bnd [hunk 5]
```diff
@@ -1,41 +0,0 @@
-#*******************************************************************************
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.ws.security/bnd.bnd:1]: Did you delete the entire bundle project or just the bnd.bnd? Only the deleted bnd.bnd is showing in the review page, so if you haven't already make sure you delete the entire project.

File: dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd (hunk 1 above)
Comment (BerksanAtes) [reply dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd:35]: sure

File: dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd (hunk 2 above)
Comment (BerksanAtes) [reply dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd:19]: This was same at com.ibm.ws.org.apache.cxf.ws.security/bnd.bnd. Should I change?

File: dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides (hunk 4 above)
Comment (BerksanAtes) [reply dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides:35]: This was like this with the original bundle. Sure I'll fix it.

**Comment by andrea-pichler** [comment]
Completed the requested L2 message review

**Comment by BerksanAtes** [comment]
#run-libby-bot

**Comment by BerksanAtes** [comment]
#build

**Comment by BerksanAtes** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat,com.ibm.ws.wssecurity_fat.cxf,com.ibm.ws.wssecurity_fat.saml,com.ibm.ws.wssecurity_fat.saml.2,com.ibm.ws.wssecurity_fat.twas

**Pull request closed without merging**